test/hex_tarball_tests.erl

```diff
@@ -18,12 +18,14 @@ disk_test() ->
     hex_test_helpers:in_tmp(fun() ->
         ok = file:write_file("foo.erl", <<"-module(foo).">>),
         ok = file:change_mode("foo.erl", 8#100644),
+        ok = file:write_file("bad.erl", <<"bad">>),
 
         Metadata = #{<<"app">> => <<"foo">>, <<"version">> => <<"1.0.0">>, <<"build_tool">> => <<"rebar3">>},
-        Files = ["foo.erl"],
+        Files = [".", "foo.erl"],
```

Comment:
"." simulates adding the whole directory; not sure why we were adding directories to list of files in the first place (in Hex client)

Reply:
I believe it is because `Path.wildcard` returns directories.